vispy/visuals/markers.py

```diff
@@ -282,24 +514,36 @@ def set_data(self, pos=None, style='o', size=10., edge_width=1.,
             The array of locations to display each symbol.
         style : str
             The style of symbol to draw (see Notes).
-        size : float
+        size : float or array
             The symbol size in px.
         edge_width : float
-            The width of the symbol outline in px.
+            The width of the symbol outline in px if larger than 1,
+            the width as a fraction of marker size if strictly smaller than 1
         edge_color : Color | ColorArray
             The color used to draw each symbol outline.
         face_color : Color | ColorArray
             The color used to draw each symbol interior.
+        scaling : bool
+            If set to True, marker scales when rezooming
+            (needs a viewbox to work).
+        small_edge_fade_out : bool
```

Comment:
@bdurin so let's get rid of this option for now (so that the API / arguments do not grow). I think that's all that's left to do before we can merge.